packages/stable/src/__tests__/api/documentsApi.int.spec.ts

```diff
@@ -0,0 +1,100 @@
+// Copyright 2022 Cognite AS
+
+import CogniteClient from '../../cogniteClient';
+import { setupLoggedInClient } from '../testUtils';
+
+describe('Documents integration test', () => {
```

Comment:
What is the canonical source of this test data, in case something gets mixed up (e.g. more files added to the same project that would break these tests)?

Reply:
Most of the tests will work regardless of what kinds of files are in the project, as long as there are a few files there. The only exception is maybe `search with query` which requires that there is at least one file with the word "test" somewhere in the project.

Reply:
These integration tests have external dependencies and assumtions that is not explicit, so it would help a lot to give a better understanding of this. Have you added new data? Are there seed data available somewhere else? What would we do in case someone wipe the project data? There is a hidden cost here. Might be there are things in the SDK that covers this, but I'm not sure since I haven't touched this here before. I can't find any documentation for integration tests in this repo from a quick look for it.